lib/Controller/Stats.php

```diff
@@ -541,6 +541,7 @@ public function export()
         $fromDt = $this->getSanitizer()->getDate('fromDt');
         $toDt = $this->getSanitizer()->getDate('toDt');
         $displayId = $this->getSanitizer()->getInt('displayId');
+        $isExport = $this->getSanitizer()->getInt('isExport', 1);
```

Comment:
We would normally use the `getCheckbox()` method here as it returns for On/1/true

```suggestion
        $isExport = $this->getSanitizer()->getCheckbox('isExport');
```